app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,29 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "devise/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
+    <% end %><br />
+    <%= f.password_field :password, autocomplete: "new-password" %>
+  </div>
+
```

Comment:
- [ ] Also I am able to sign up but after that, it says `you'll get an email notification to confirm your email address` but notification never comes and because of this I am unable to go to the other pages after `sign-up` or `sign-in`.

![Screenshot from 2022-01-05 02-07-48](https://user-images.githubusercontent.com/22434660/148121163-e25087d2-7f7c-429b-a428-2470f20665fa.png)

Reply:
Thanks for note this, now the bug it's fixed and you can login after the registration 😄